src/parsing/parsers/clickhouse/parser-extension.js

```diff
@@ -1298,6 +1298,48 @@ export const extendParser = function (parser) {
     parser.yy.result.suggestTemplates = true;
   }
 
+  parser.suggestEngines = function() {
```

Comment:
Actually we need to make revision of suggested engines, currently I included all engines which `DataGrip` supports, except PostgreSQL.